core/types/receipt_test.go

```diff
@@ -966,26 +966,26 @@ func TestDeriveOptimismEcotoneTxReceipts(t *testing.T) {
 	diffReceipts(t, receipts, derivedReceipts)
 }
 
-func TestDeriveOptimismHoloceneTxReceipts(t *testing.T) {
-	// Holocene style l1 attributes with baseFeeScalar=2, blobBaseFeeScalar=3, baseFee=1000*1e6, blobBaseFee=10*1e6, operatorFeeScalar=7, operatorFeeConstant=9
+func TestDeriveOptimismIsthmusTxReceipts(t *testing.T) {
+	// Isthmus style l1 attributes with baseFeeScalar=2, blobBaseFeeScalar=3, baseFee=1000*1e6, blobBaseFee=10*1e6, operatorFeeScalar=7, operatorFeeConstant=9
 	payload := common.Hex2Bytes("d1fbe15b000000020000000300000000000004d200000000000004d200000000000004d2000000000000000000000000000000000000000000000000000000003b9aca00000000000000000000000000000000000000000000000000000000000098968000000000000000000000000000000000000000000000000000000000000004d200000000000000000000000000000000000000000000000000000000000004d2000000070000000000000009")
```

Comment:
I think the payload needs to change for the isthmus signature. See l1attributes.md in the sepc